### PR TITLE
Small cleanups

### PR DESCRIPTION
**Pull request by AlexBenton**
Simplify the chapter 5 examples a little bit
Pull out RiftManager and RiftGlfwApp to their own CPP files
Clear up duplicated code in OvrUtils
Add a grid to the demos for 5 (Not sure if I like this effect--it's a little too white, looks garish maybe?)

### CODE REVIEW

File: examples/cpp/Example_5_3_RiftRendered.cpp [hunk 1]
```diff
@@ -10,8 +10,7 @@ class CubeScene_Rift: public RiftGlfwApp {
   PerEyeArg eyes[2];
   ovrTexture eyeTextures[2];
 
-  float ipd{ OVR_DEFAULT_IPD };
-  float eyeHeight{ OVR_DEFAULT_EYE_HEIGHT };
+  float ipd, eyeHeight;
```
Comment (jherico) [inline examples/cpp/Example_5_3_RiftRendered.cpp]: what's the purpose of removing the defaults here?

File: examples/cpp/Example_5_5_RiftTimewarp.cpp [hunk 2]
```diff
@@ -1,128 +0,0 @@
-#include "Common.h"
```
Comment (jherico) [inline examples/cpp/Example_5_5_RiftTimewarp.cpp]: Why was this file removed?

File: examples/cpp/common/Utils.h [hunk 3]
```diff
@@ -41,10 +41,6 @@ class RateCounter {
     times.clear();
   }
 
-  unsigned int count() const {
```
Comment (jherico) [inline examples/cpp/common/Utils.h]: I don't get why you would remove this.  Knowing the count of measured frames might be valuable to a caller.

File: examples/cpp/Example_5_3_RiftRendered.cpp (hunk 1 above)
Comment (AlexBenton) [inline examples/cpp/Example_5_3_RiftRendered.cpp]: The defaults are useless (and therefore distracting.)  The member fields
are assigned values immediately in the constructor.

File: examples/cpp/Example_5_5_RiftTimewarp.cpp (hunk 2 above)
Comment (AlexBenton) [inline examples/cpp/Example_5_5_RiftTimewarp.cpp]: Because it now only differs from Example_5_4 by a single line.  When this
example was in Chapter 5, it made sense to contrast it against 5_4; but now
that it's in Chapter 6, there's no reason to keep it as complex as it is to
compare it to the previous demo.  Once I eliminated all the complexity, I
was left with RiftApp.  There wasn't anything left in this file.

File: examples/cpp/common/Utils.h (hunk 3 above)
Comment (AlexBenton) [inline examples/cpp/common/Utils.h]: The method is never called in our code.
